guide/popular-topics/components-v2.md

```diff
@@ -0,0 +1,230 @@
+
+In the following section, we will explain what kind of components are new, how they work together with interactive components, and the limitations Discord has set when using CV2 components in your message.
+
+## New components
+
+CV2 brought new layout components and some new content components. The content components are very similar to some elements used in embeds, but their behaviour is slightly different when used relative to their use in embeds.
+
+### Text Display
+
+A Text Display is a content component allowing you to add text to your message formatted with Discord's available markdown. This component is very similar to the `content` field of a message, but using multiple of these components you can control the layout of your message. You can use the <DocsLink path="TextDisplayBuilder:Class" /> utility class to easily create a Text Display component.
+
+::: tip
+In addition of the available markdown in Text Display components, you can also use the mention of users and roles, which actually notifies these users and roles, unlike the mentions used in embeds! You can always add the `allowedMentions` field to your message payload to control who will be notified.
+:::
+
+The example below shows how you can send a Text Display component in a channel.
+
+```js
+const { TextDisplayBuilder, MessageFlags } = require('discord.js');
+
+const exampleTextDisplay = new TextDisplayBuilder()
+	.setContent('This text is inside a new Text Display component! You can use **any __markdown__** available inside this component too.');
+
+await channel.send({
+	components: [exampleTextDisplay],
+	flags: MessageFlags.IsComponentsV2,
+});
+```
+
+### Thumbnail
+
+A Thumbnail is a content component that is visually similar to the `thumbnail` field inside an embed, but with CV2, it can only be added as an accessory inside a [Section](/popular-topics/components-v2.md#section) component. However, you can add alt text to the image as well as marking the image as a spoiler. You can use the <DocsLink path="ThumbnailBuilder:Class" /> utility class to easily create a Thumbnail component.
+
+The example below shows how you can send a Thumbnail component as an Section component accessory in a channel.
+
+```js
+const { AttachmentBuilder, SectionBuilder, TextDisplayBuilder, ThumbnailBuilder, MessageFlags } = require('discord.js');
+
+const file = new AttachmentBuilder('../assets/image.png');
+
+const exampleSection = new SectionBuilder()
+	.addTextDisplayComponents(
+		new TextDisplayBuilder()
+			.setContent('This text is inside a new Text Display component! You can use **any __markdown__** available inside this component too.'),
+	)
+	.setThumbnailAccessory(
+		new ThumbnailBuilder()
```

Comment:
```suggestion
		(thumbnail) => thumbnail
```
And similarly for all other places `MediaGallery`, `Section` or `Container` get other builders added.